WooCommerce/Classes/ViewRelated/Dashboard/Settings/Settings/SettingsViewModel.swift

```diff
@@ -361,6 +361,17 @@ private extension SettingsViewModel {
 // MARK: - Localizations
 //
 private extension SettingsViewModel {
+    enum IPPPluginStatus {
+        static let stripe_plugin_slug = "woocommerce-gateway-stripe/woocommerce-gateway-stripe"
+        static let wcpay_plugin_slug = "woocommerce-payments/woocommerce-payments"
+        static let woo_mobile_stripe_not_installed = "woo_mobile_stripe_not_installed"
+        static let woo_mobile_stripe_installed_and_not_activated = "woo_mobile_stripe_installed_and_not_activated"
+        static let woo_mobile_stripe_installed_and_activated = "woo_mobile_stripe_installed_and_activated"
+        static let woo_mobile_wcpay_not_installed = "woo_mobile_wcpay_not_installed"
+        static let woo_mobile_wcpay_installed_and_not_activated = "woo_mobile_wcpay_installed_and_not_activated"
+        static let woo_mobile_wcpay_installed_and_activated = "woo_mobile_wcpay_installed_and_activated"
+        static let woo_mobile_site_plugins_fetching_error = "woo_mobile_site_plugins_fetching_error"
+    }
```

Comment:
```suggestion
```
This needs to be removed as [is declared](https://github.com/woocommerce/woocommerce-ios/blob/9a1ae44d69d7f4bd509301cd3b40487d033a8f0a/WooCommerce/Classes/Tools/Zendesk/ZendeskManager.swift#L1120-L1130) on ZendeskManager now, I forgot 😬

Reply:
Fixed 849862e0e101050badcb6e53a96490ae3bc31dc8